modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -1473,8 +1485,7 @@ private DTOConverterContext _getDTOConverterContext(
 			Map<String, Serializable> attributes)
 		throws PortalException {
 
-		GraphQLContext graphQLContext =
-			dataFetchingEnvironment.getLocalContext();
+		GraphQLContext graphQLContext = dataFetchingEnvironment.getContext();
```

Comment:
?

Reply:
I've checked again and adding the namespace makes getLocalContext to be null. I think it is because the request is not local anymore as it goes one level deeper into the namespace Data Fetcher

Reply:
👍